exercises/anagram/examples/list-string/package.yaml

```diff
@@ -0,0 +1,17 @@
+name: anagram
```

Comment:
The advantage would be... fewer package.yaml to have to keep up to date if we have to change them all?

the disdavntage would be that the test code would get slightly more complex.